### ISSUE TITLE
WP8 - List-Item-Button clicks not working

### ISSUE THREAD

**Issue by graphefruit** [angular/material#7618]
Hello Team,

with the latest version the button click isn't working anymore in WP8 on in list items

The problem seems like that the two-button rendering causing wired issues in WP.
```
<md-list-item aria-label="filter" no-padding ng-click="doSomething()">
   <p>Filter</p>
   <md-icon md-svg-icon="image:navigate_next"></md-icon>
</md-list-item>
```
The render output looks like: 
```
<md-list-item role="listitem" class="_md-button-wrap md-clickable" tabindex="-1">
    <button class="md-button _md-no-style">
        <button class="_md-no-style md-button" type="button" ng-transclude="" md-no-focus-style=""  ng-click="doSomething()">
        </button>
      <div class="_md-list-item-inner">
          <div layout="column" class="layout-column">
           ...
          </div>
      </div>
   </button>
 </md-list-item>
```

Update: I think the problem is that the outer button triggers but not the inner button, and the event is lost cause of this.

Greetings & Thanks
Graphefruit

**Comment by DevVersion** [comment]
Hey Graphefruit (+1 name),

Can you confirm your bug with the latest version? I've recently removed the outer button, because it was accidentally added by me. (should be a `DIV`)

**Comment by graphefruit** [comment]
Hey @DevVersion thanks for helping me out. I'll get in touch with you tomorrow. 
Greetings

**Comment by ThomasBurleson** [comment]
Latest version [https://material.angularjs.org/HEAD/demo/list](https://material.angularjs.org/HEAD/demo/list) has this working properly:

![screen shot 2016-03-17 at 12 17 19 pm](https://cloud.githubusercontent.com/assets/210413/13854601/478ffa2a-ec3a-11e5-9482-a58918d98d89.png)

**Comment by graphefruit** [comment]
@ThomasBurleson this works in WP aswell as in IE-Emulation. PS: The animations and events are very delayed and laggy in WP. In iOS and Android they work fine.
@DevVersion is this already your changed version there?

**Comment by DevVersion** [comment]
@graphefruit Yes, that's the latest version.

**Comment by graphefruit** [comment]
I did a "small" video: https://vid.me/tUyZ this is how it looks and feels like.
In my project I came up to the solution to disable nearly all transitions and all ripple effects I could find.
PS: If this is overloading this topic just tell me and I gonna create a new one so.

**Comment by DevVersion** [comment]
@graphefruit Thanks for the video, but I would like to retrieve more information, because I can't reproduce this. 

- What's your browser?
- Is this running in an Android WebView (ChromeWebAdapter, or Default)?
- Are you using Cordova?

In addition to that, I would like to have a demo with source, like a codepen.

**Comment by graphefruit** [comment]
Hello @DevVersion this video shows a normal Nokia Lumia with Windows 8.1 and its default browser.
The problem is that just an emulator or a real lumia device will show you this problems.

So to your questions in a summary:
- Windows Phone standard browser
- No Android Webview
- No Cordova

If you need more details just let me know.

**Comment by DevVersion** [comment]
@graphefruit Just tested it on a Nokia Lumia 930 with Win 8.1 on Browserstack. I can't reproduce your issue. 

In addition, the button inside of the dialog isn't related in any way to the list component. 

So as conclusion, I can say that this isn't caused by the list component, it must be a performance issue and seems to be user specific.

**Comment by graphefruit** [comment]
@DevVersion I'm using a Lumia 620 at work. This just has a 1 dual core GHz processor. The 930 instead has a 2.2 quad core processor. So I think there is a huge difference. 

Like I said this topic isn't related to the created one. So maybe I should make another one with Windows Phone performance issue? 

The button seems to work after your fix like I proved. 

This conversation is about laggy animations and events. 

May you have the chance to get an older device to test or maybe a real one? 

The simulators are not always as bad as the real devices, I already noticed this my self. The simulators mostly doesn't bring up those errors. 

Also I could try to set up a live meeting to demonstrate it via the real device. 

At all thanks for you help in advance.
